frontend/src/routes/router.tsx

```diff
@@ -54,9 +54,12 @@ const StudentClassIndex = lazy(
 );
 
 // ==== LEARNING PATHS ==== //
+
```

Comment:
```suggestion
import LocalLearningObjectsPage from '@/pages/teacher/LocalLearningObject';
```
Deze import moet er ook nog bij anders laadt de website niet.